src/pages/admin/materiaalaanvragen/index.tsx

```diff
@@ -1,25 +1,275 @@
+import {
+	Dropdown,
+	MenuContent,
+	MenuItemInfo,
+	OrderDirection,
+	Table,
+} from '@meemoo/react-components';
+import clsx from 'clsx';
+import { isEmpty, isNil } from 'lodash';
 import { GetServerSidePropsResult, NextPage } from 'next';
 import { GetServerSidePropsContext } from 'next/types';
-import React, { ComponentType } from 'react';
+import React, { ComponentType, MouseEvent, ReactNode, useMemo, useState } from 'react';
+import { Row, SortingRule, TableState } from 'react-table';
+import { useQueryParams } from 'use-query-params';
 
+import MaterialRequestDetailBlade from '@account/components/MaterialRequestDetailBlade/MaterialRequestDetailBlade';
 import { Permission } from '@account/const';
+import {
+	ADMIN_MATERIAL_REQUESTS_QUERY_PARAM_CONFIG,
+	ADMIN_MATERIAL_REQUESTS_TABLE_PAGE_SIZE,
+	getAdminMaterialRequestTableColumns,
+} from '@admin/const/material-requests.const';
 import { AdminLayout } from '@admin/layouts';
 import { withAuth } from '@auth/wrappers/with-auth';
+import {
+	CP_MATERIAL_REQUEST_TYPE_FITLER_ARRAY,
+	CP_MATERIAL_REQUEST_TYPE_FITLER_RECORD,
+	CP_MATERIAL_REQUESTS_FILTER_ALL_ID,
+} from '@cp/const/material-requests.const';
+import { useGetContentPartners } from '@cp/hooks/get-content-partners';
+import {
+	Icon,
+	IconNamesLight,
+	Loading,
+	PaginationBar,
+	SearchBar,
+	sortingIcons,
+} from '@shared/components';
 import PermissionsCheck from '@shared/components/PermissionsCheck/PermissionsCheck';
+import { globalLabelKeys } from '@shared/const';
 import { getDefaultServerSideProps } from '@shared/helpers/get-default-server-side-props';
 import { renderOgTags } from '@shared/helpers/render-og-tags';
 import useTranslation from '@shared/hooks/use-translation/use-translation';
 import { DefaultSeoInfo } from '@shared/types/seo';
 
+import { useGetMaterialRequestById } from '@material-requests/hooks/get-material-request-by-id';
+import { useGetMaterialRequests } from '@material-requests/hooks/get-material-requests';
+import {
+	MaterialRequest,
+	MaterialRequestKeys,
+	MaterialRequestType,
+} from '@material-requests/types';
+
 const AdminMaterialRequests: NextPage<DefaultSeoInfo> = ({ url }) => {
-	const { tText } = useTranslation();
+	const { tHtml, tText } = useTranslation();
+	const [isTypeDropdownOpen, setIsTypeDropdownOpen] = useState(false);
+	const [isMaintainerDropdownOpen, setIsMaintainerDropdownOpen] = useState(false);
+	const [typeDropdownLabel, setTypeDropdownLabel] = useState(
```

Comment:
Check out `CP_MATERIAL_REQUEST_TYPE_FITLER_RECORD` in `src/modules/cp/const/material-requests.const.tsx`. By implementing this in the same way as in the other file, it will be easier to make adjustments later since they will both "work" the same way ;)